frontend/apps/client/src/store/home/index.ts

```diff
@@ -0,0 +1,20 @@
+import { atom } from 'jotai';
+import { atomFamily } from 'jotai/utils';
+
+import type { OngoingScheduleAttendType } from '@/features/shared-schedule/model';
+
+export const ongoingSegmentValueAtom = atom<OngoingScheduleAttendType>('ALL');
+
+export interface OngoingSegmentStateAtom {
+  listIndex: number;
+  page: number;
+}
+
+export const ongoingSegmentAtomFamily = atomFamily((_: OngoingScheduleAttendType) => 
+  atom({
+    listIndex: 0,
+    page: 1,
+  } as OngoingSegmentStateAtom),
```

Comment:
p3;
단언문보다는 제네릭 활용이 더 안정성 있을 것 같습니다!
```suggestion
  atom<OngoingSegmentStateAtom>({
    listIndex: 0,
    page: 1,
  }),
```

Reply:
넵 좋습니다!